Sample Applications/WPFGallery/Views/Navigation/MenuPage.xaml

```diff
@@ -38,64 +38,67 @@
-                    <Menu FontSize="14">
-                        <MenuItem Header="File">
-                            <MenuItem Header="New" />
-                            <MenuItem Header="New window" />
-                            <MenuItem Header="Open..." />
-                            <MenuItem Header="Save" />
-                            <MenuItem Header="Save As..." />
+                    <StackPanel>
+                        <TextBlock x:Name="StatusMenuItem" Text="" />
+                        <Menu FontSize="14">
+                            <MenuItem Header="File">
+                                <MenuItem Header="New" Click="MenuItem_Click" />
+                                <MenuItem Header="New window" Click="MenuItem_Click" />
+                                <MenuItem Header="Open" Click="MenuItem_Click" />
+                                <MenuItem Header="Save" Click="MenuItem_Click" />
+                                <MenuItem Header="Save As" Click="MenuItem_Click" />
+                                <Separator />
+                                <MenuItem Header="Exit" />
+                            </MenuItem>
+                            <MenuItem Header="Edit">
+                                <MenuItem Header="Undo" Click="MenuItem_Click" />
+                                <Separator />
+                                <MenuItem Header="Cut" Click="MenuItem_Click" />
+                                <MenuItem Header="Copy" Click="MenuItem_Click" />
+                                <MenuItem Header="Paste" Click="MenuItem_Click" />
+                                <MenuItem IsEnabled="False" />
+                                <Separator />
+                                <MenuItem Header="Search with browser" Click="MenuItem_Click" />
+                                <MenuItem Header="Find" Click="MenuItem_Click" />
+                                <MenuItem Header="Find next" Click="MenuItem_Click" />
+                                <Separator />
+                                <MenuItem Header="Select All" />
+                            </MenuItem>
```

Comment:
Rather than adding Click="MenuItem_Click" in all the menu items, create a local style as StackPanel.Resources extending the default menu item style, it will reduce redundancy in the code.